react/features/shared-iframe/functions.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import { getFakeParticipants } from '../base/participants';
+import { toState } from '../base/redux';
+
+import { IFRAME_PLAYER_PARTICIPANT_NAME } from './constants';
+
+
+/**
+ * Returns true if an IFrame is shared in the meeting.
+ *
+ * @param {Object | Function} stateful - The Redux state or a function that gets resolved to the Redux state.
+ * @returns {boolean}
+ */
+export function isIFrameSharingActive(stateful: Object | Function): boolean {
+
+    let isIFrameActive = false;
+    const state = toState(stateful);
+    const { sharedIFrameName } = state['features/base/config'];
+
+    // eslint-disable-next-line no-unused-vars
```

Comment:
How do you mean? It is destructuring and naming the `Map<string, Participant>` return value of the function. I don't see how Object.values would help. So I am interested in what you meant! 
```js
for (const participant of getFakeParticipants(stateful)) {
        if (participant[1].name === sharedIFrameName || participant[1].name === IFRAME_PLAYER_PARTICIPANT_NAME) {
            isIFrameActive = true;
            break;
        }
    }
```

Reply:
Ah I thought it was an object! In that case you can do `theMap.values()` and you'll get an iterator over the values, which is what you need.

Reply:
I tried it with:
```
for (const { p } of getFakeParticipants(stateful).values()) {
    if (p.name === sharedIFrameName || p.name === IFRAME_PLAYER_PARTICIPANT_NAME) {
        isIFrameActive = true;
        break;
    }
}
```
But this will throw an error property not found on `undefined`.